app/database/models/section.py

```diff
@@ -40,3 +40,6 @@ def add_video(self, video: "VideoModel") -> None:
         self.videos.append(video)
         db.session.add(self)
         db.session.commit()
+
+    def json(self):
```

Comment:
this is not the correct way to create a JSON object. You can use jsonify after serializing the dictionary object.

Reply:
I would suggest using the mapping function `map_to_dto` to follow the pattern (convert Python model object to representation object, which can be different for different endpoints/scenarios).